cmd/manager/main.go

```diff
@@ -58,6 +91,55 @@ func main() {
 		fatal(err)
 	}
 
+	// Check if storageos scheduler should be enabled.
+	enableScheduler := false
```

Comment:
Rather than force the var to be set, should we set the default to enabled and have an optional `DISABLE_SCHEDULER` instead?  It's good to expose options, but I think the defaults should be the recommended config? 

In this case it would also match the CR's `disableScheduler` field.

Reply:
Yes, that would be better, else it can become confusing.

Reply:
Changing the env var name to `DISABLE_SCHEDULER_WEBHOOK` to make it more clear. Not to confuse with the actual scheduler.

Reply:
Thanks Sunny, LGTM!